src/NuGet.Licenses/App_Data/Licenses/389-exception.json

```diff
@@ -1,6 +1,6 @@
+    "html":  "\n      \u003cp\u003eThis Program is free software; you can redistribute it and/or\n       modify it under the terms of the GNU General Public License as\n       published by the Free Software Foundation; version 2 of the\n       License.\u003c/p\u003e\n\n      \u003cp\u003eThis Program is distributed in the hope that it will be useful,\n       but WITHOUT ANY WARRANTY; without even the implied warranty of\n       MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the\n       GNU General Public License for more details.\u003c/p\u003e\n\n      \u003cp\u003eYou should have received a copy of the GNU General Public License\n       along with this Program; if not, write to the Free Software\n       Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA\n       02111-1307 USA.\u003c/p\u003e\n\n      \u003cp\u003eIn addition, as a special exception, Red Hat, Inc. gives You the\n       additional right to link the code of \u003cvar class=\"replaceable-license-text\"\u003e this Program\u003c/var\u003e with code\n       not covered under the GNU General Public License (\u0026quot;Non-GPL\n       Code\u0026quot;) and to distribute linked combinations including the\n       two, subject to the limitations in this paragraph. Non-GPL\n       Code permitted under this exception must only link to the code\n         of \u003cvar class=\"replaceable-license-text\"\u003e this Program\u003c/var\u003e through those well defined interfaces\n       identified in the file named EXCEPTION found in the source\n       code files (the \u0026quot;Approved Interfaces\u0026quot;). The files of Non-GPL\n       Code may instantiate templates or use macros or inline\n       functions from the Approved Interfaces without causing the\n       resulting work to be covered by the GNU General Public\n       License. Only Red Hat, Inc. may make changes or additions to\n       the list of Approved Interfaces. You must obey the GNU General\n       Public License in all respects for all of the \u003cvar class=\"replaceable-license-text\"\u003e Program\u003c/var\u003e code and\n       other code used in conjunction with \u003cvar class=\"replaceable-license-text\"\u003e the Program\u003c/var\u003e except the\n       Non-GPL Code covered by this exception. If you modify this\n       file, you may extend this exception to your version of the\n       file, but you are not obligated to do so. If you do not wish\n       to provide this exception without modification, you must\n       delete this exception statement from your version and license\n       this file solely under the GPL without exception.\u003c/p\u003e\n\n    ",
```

Comment:
@fjssilva 
Could we revert back those changes existing files unless there was actual typo or mismatch? Change in existing files makes really hard to review this PR, I just compared 1 of them only change seems like new line. I feel making unnecessary changes  many documents same time is risk.

Reply:
Mind that I didn't change these files myself. This was done automatically by the script [license-text-extractor.ps1](https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1) which downloads all licence files from https://spdx.org/licenses/ and re-creates local copies from them by extracting the corresponding sections from the original HTML. If there are changes, then this is because the license files were changed in the origin.

Reply:
FYI, I only fixed a bug in the script [license-text-extractor.ps1](https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1) and follow the instructions [Updating license text](https://github.com/NuGet/NuGet.Licenses?tab=readme-ov-file#updating-license-text).

Reply:
Please revert this change and create separate PR if necessary.